content/run_example_workflow.py

```
@@ -47,13 +51,38 @@
+        mzmlfileworkflow.result_section(result_dir)
 
+st.write("## Hardware utilization")
+cpu_ram_stats_placeholder = st.empty()
 
-mzmlfileworkflow.result_section(result_dir)
\ No newline at end of file
+# in this continuous loop we measure CPU/RAM metrics and
+# monitor the state of the active workflow job
+while True:
+    # monitor if active job exists
+    if st.session_state['mzml_workflow_job_id'] != None:
```

Ruff (0.8.2)

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)